scripts/tools/tools-object.js

```diff
@@ -65,8 +65,10 @@ async function convertTools(data) {
           }
         });
       } else {
-        console.log("Repository: " + tool.repository.html_url)
-        console.log("Error: " + validate.errors)
+        console.error('Invalid .asyncapi-tool file.');
+        console.error(`Located in: ${tool.html_url}`);
```

Comment:
Yeah, from my perspective, certain users are making `.asyncapi-tool` files inside any subdirectories. If we want to locate the wrong file, it may occur that we get into trouble finding the file inside the repository sub-directories. Hence, using `tool.html_url` is the best link to directly hop into the file in the respective repository to see the errors, instead console the repository's URL. WDYT @magicmatatjahu @derberg ?

Reply:
@akshatnema you are right, `tool.html_url` provides direct URL to the file

@magicmatatjahu changed

folks, please approve

Reply:
Sorry, I was looking at the code and we used a different field. Approved :)